main.cpp

```diff
@@ -52,114 +52,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun: plane
-//  action 1: flies
-plane.fly();
+//  Noun: airPlane
+//  action 1: turns around
+airPlane.turnsAround();
 
-//  action 2: crashes
-plane.crash();
+//  action 2: crashes and burns
+airPlane.crashAndBurn();
 
-//  action 3: takes off
-plane.takeOff();
+//  action 3: executes a nose dive
+airPlane.executeNoseDive();
 
 //  2)
-//  Noun: cat
-//  action 1: yawns
-cat.yawn();
+//  Noun: saberTooth
+//  action 1: rolls around
+saberTooth.rollAround();
 
-//  action 2: jumps
-cat.jump();
+//  action 2: hides in bushes
+saberTooth.hideInBush();
 
-//  action 3: runs
-cat.run();
+//  action 3: surprise attacks prey
+saberTooth.surpriseAttack();
  
 //  3)
-//  Noun: bird
-//  action 1: sings
-bird.sing();
+//  Noun: redParrot
+//  action 1: mimics words
+redParrot.mimicWord();
 
 //  action 2: flaps its wings
-bird.flapWings();
+redParrot.flapWings();
 
-//  action 3: dies
-bird.die();
+//  action 3: screams horribly
+redParrot.screamHorribly();
  
 //  4)
-//  Noun: firetruck
-//  action 1: puts out fire
-fireTruck.putOutFire();
+//  Noun: policeOfficer
+//  action 1: chases criminals
+policeOfficer.chaseCriminal();
 
-//  action 2: extends laddders
-fireTruck.extendLadder();
+//  action 2: tackles criminals
+policeOfficer.tackleCriminal();
 
 //  action 3: sounds alarm
-fireTruck.soundAlarm();
+policeOfficer.soundAlarm();
  
 //  5)
-//  Noun: soldier
-//  action 1: crawls
-soldier.crawl();
+//  Noun: footSoldier
+//  action 1: takes cover
+footSoldier.takeCover();
 
-//  action 2: shoots
-soldier.shoot();
+//  action 2: shoots guns
+footSoldier.shootGun();
 
-//  action 3: hides
-soldier.hide();
+//  action 3: throws grenades
+footSoldier.throwGrenade();
 
 //  6)
-//  Noun: wasp
-//  action 1: stings
-wasp.sting();
+//  Noun: killerBee
+//  action 1: sting the enemy
+killerBee.stingEnemy();
 
-//  action 2: flies
-wasp.fly();
+//  action 2: takes flight
+killerBee.takeFlight();
 
 //  action 3: scares everyone and everything
-wasp.scareEveryoneAndEverything();
+killerBee.scareEveryoneAndEverything();
  
 //  7)
-//  Noun: oven
+//  Noun: heatSink
 //  action 1: heats up
-oven.heatup();
+heatSink.heatUp();
 
-//  action 2: consumes energy
-oven.consumeEnergy();
+//  action 2: exchanges heat
+heatSink.exchangeHeat();
 
-//  action 3: bakes food
-oven.bakeFood();
+//  action 3: protects device from over heating
+heatSink.protectDeviceFromHeat();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.
Once you have completed the above requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.